cfgov/unprocessed/js/molecules/GlobalSearch.js

```diff
@@ -17,19 +19,23 @@ var breakpointState = require( '../modules/util/breakpoint-state' );
  *   The DOM element within which to search for the molecule.
  * @returns {Object} An GlobalSearch instance.
  */
-function GlobalSearch( element ) {
+function GlobalSearch( element ) { // eslint-disable-line max-statements, no-inline-comments, max-len
 
   var BASE_CLASS = 'm-global-search';
 
   var _dom =
     atomicCheckers.validateDomElement( element, BASE_CLASS, 'GlobalSearch' );
-  var _triggerDom = _dom.querySelector( '.' + BASE_CLASS + '_trigger' );
+  var _triggerSel = '.' + BASE_CLASS + '_trigger';
+  var _triggerDom = _dom.querySelector( _triggerSel );
+  var _flyoutMenu = new FlyoutMenu( _dom,
+                                    _triggerSel,
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/b23f666702bb8874c9b9e7daf93185075dd1a000